frontend/src/component/Search/SpaceFacets.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import CheckboxFilter from './CheckboxFilter.jsx'
+import { SEARCH_SPACE_FACETS } from '../../util/helper.js'
+
+export class SpaceFacets extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      showMemberList: true,
+      showAuthorList: true
+    }
+  }
+
+  handleOpenOrCloseFilter = (filter) => {
+    switch (filter) {
+      case SEARCH_SPACE_FACETS.MEMBER.slug:
+        this.setState(prev => ({ showMemberList: !prev.showMemberList }))
+        break
+      case SEARCH_SPACE_FACETS.AUTHOR.slug:
+        this.setState(prev => ({ showAuthorList: !prev.showAuthorList }))
+        break
+    }
+  }
```

Comment:
Wouldn't it be easier to have two functions? As you know the facet type when you create it?